platforms/org.eclipse.mita.platform.arduinouno/src/org/eclipse/mita/platform/arduinouno/platform/EventLoopGenerator.xtend

```diff
@@ -0,0 +1,55 @@
+package org.eclipse.mita.platform.arduinouno.platform
```

Comment:
This is an interesting design of an event loop which would really need some in-depth documentation. Implementers would benefit from understanding how this works (one loop with all handlers statically linked and turned on/off with flags) and how it contrasts with e.g. the XDK (queue where function pointers get inserted and executed by a separate task one after the other).

The question of how to build the event loop comes up rather often, so this could become a really good example.